friendship-quiz.js

```diff
@@ -1,3 +1,5 @@
+//comment for pull request
+
```

Comment:
Great code, Blessing! I ***love*** your use of a for-loop to check if each `guess` is in `answerList`, and your use of the boolean `found` to hold whether or not `guess` has been found in `answerList` yet. 
I have two comments:
- `console.log()` is a ***great*** function to use while you're programming, but in the end, the user won't actually be interacting with the browser console. These lines are helpful to you while you're debugging, but can be deleted when cleaning up your code to create the final product! 
- I would've liked a helpful comment explaining what the `found` variable is for!

Great work 😎